app/javascript/components/rooms/RoomCard.jsx

```diff
@@ -10,24 +10,18 @@ export default function RoomCard(props) {
   const handleClick = useCallback(() => { navigate(id); }, [id]);
 
   return (
-    <Container>
-      <Card id="rooms-card" style={{ width: '14rem' }} border="dark">
-        <Card.Body id="room-card-top" onClick={handleClick}>
-          <PersonSquare size={30} />
-          <Card.Title>
-            {' '}
-            {name}
-            {' '}
-          </Card.Title>
-          {/* TODO: Hadi- Make last session dynamic per room */}
-          <Card.Text> Last session... </Card.Text>
-        </Card.Body>
-        <Card.Body>
-          <hr />
-          <Link45deg id="clipboard-icon" size={20} />
-          <ButtonLink className="float-end" to="#"> Start</ButtonLink>
-        </Card.Body>
-      </Card>
-    </Container>
-  );
+    <Card className='rooms-card' border='light'>
+      <Card.Body className='room-card-top pb-0' onClick={handleClick}>
+        <PersonSquare className='mb-4' size={55}/>
+        <Card.Title> {name} </Card.Title>
+        {/* TODO: Hadi- Make last session dynamic per room */}
+        <Card.Text className='text-muted'> Last session... </Card.Text>
+        <hr />
+      </Card.Body>
+      <Card.Body className='pt-0'>
```

Comment:
Why having two Card bodies?
Should it be either inside the `Card.body` or in a `Card.Footer`?
Is omitting  the use of `Card.header` and `Card.Footer` because of design implications?